packages/api/src/entity/subscription.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm'
+import { User } from './user'
+import { SubscriptionStatus } from '../generated/graphql'
+
+@Entity({ name: 'subscriptions' })
+export class Subscription {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string
+
+  @ManyToOne(() => User)
+  @JoinColumn({ name: 'user_id' })
+  user!: User
+
+  @Column('text')
+  name!: string
+
+  @Column('enum', {
+    enum: SubscriptionStatus,
+    default: SubscriptionStatus.Active,
+  })
+  status!: SubscriptionStatus
+
+  @Column('text', { nullable: true })
+  description?: string
+
+  @Column('text', { nullable: true })
+  url?: string
+
+  @Column('text', { nullable: true })
+  unsubscribeMailTo?: string
+
+  @Column('text', { nullable: true })
+  unsubscribeHttpUrl?: string
+
+  @CreateDateColumn()
+  createdAt!: Date
+
+  @UpdateDateColumn()
+  updatedAt!: Date
```

Comment:
Will `updatedAt` when a new page is received for a subscription? I think we will want people to be able to order their subscription by last issue received. Also probably want to expose it in the UI so they know when they last received an article from a subscription.

Reply:
Yes we can do that